tests/test_publishing.py

```diff
@@ -169,24 +172,30 @@ def test_save_publishing_error_when_postoffice_raises_connection_error(
 @pytest.mark.django_db
 class TestBulkPublishing:
 
-    @patch('postoffice_django.publishing.requests.post')
     def test_bulk_messages_endpoint_called_when_calling_bulk_publish(
-            self, post_mock):
+            self, postoffice_valid_response):
+
+        responses.add(responses.POST,
+                      POSTOFFICE_PUBLISH_MESSAGE_URL,
+                      status=201,
+                      body=postoffice_valid_response,
+                      content_type='application/json')
+
         bulk_publish(
             topic='some_topic',
             payload=[{'key': 'key_1'}, {'key': 'key_2'}],
             hive='vlc1',
         )
+        request = responses.calls[0].request
 
-        post_mock.assert_called_with(
-            'http://fake.service/api/bulk_messages/',
-            json={
-                'topic': 'some_topic',
-                'payload': [{'key': 'key_1'}, {'key': 'key_2'}],
-                'attributes': {'hive': 'vlc1'}
+        assert json.loads(request.body) == {
+            'topic': 'some_topic',
+            'payload': [{'key': 'key_1'}, {'key': 'key_2'}],
+            'attributes': {
+                'hive': 'vlc1'
             },
-            timeout=1.2
-        )
+        }
+        assert request.headers['Content-Type'] == 'application/json'
```

Comment:
This test explicitly tests that the method `bulk_publish` calls the `/api/bulk_messages/` endpoint, and with this change it won't do that anymore. Furthermore, it's sending the request to the "normal" endpoint for single messages, so I'm not sure that this is what we want to test here.

Reply:
Totally agree with you Xavi, we are going to fix this.

Reply:
I have chosen to reset the PR to a point where we only do a soft refactor and adapt the payload to the new format.